middleware.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+)
+
+func CORSHandler(next http.Handler) http.Handler {
```

Comment:
Not currently used. It is here for two reasons:
1) Some may want to deal with CORS issues this way by just adding the handler like this inside of routes.go
`middleware = alice.New(CORSHandler)`
2) Demonstrates how to create middleware. A dev will just copy and paste to create a new middleware to be used in routes.go